__tests__/spec/utils.js

```diff
@@ -67,7 +67,7 @@ function _mkReleaseArchiveSync ({ archive, prefix }) {
   const ref = child_process.execSync('git stash create', { cwd: repoDir, encoding: 'utf8' }) || 'HEAD'
 
   child_process.execSync(
-    `git archive --worktree-attributes --prefix=${prefix}/ --output=${archive} ${ref}`,
+    `node scripts/create-release-archive --releaseName ${getReleaseVersion()} --destDir ${path.dirname(archive)} ${ref}`,
```

Comment:
Should we use `./scripts/create-release-archive.js` (and maybe remove the .js from the file itself) so it's treated more like a shell script?

Reply:
I'm not too picky, although if we want to make this change I think we might want to try and be consistent with other existing scripts (like in `lib/build`)

Reply:
Cool, we can look at that later.

Reply:
I'm happy to look at it now, might as well change it now if we want to, then we won't have inconsistencies between releases. The scripts in `lib/build` have not ever been in a release before.

Reply:
I've made the change in [2fa019a](https://github.com/alphagov/govuk-prototype-kit/pull/1402/commits/2fa019ab3483c0b5e747aa0060e272493d9edf98).